packages/nextjs/app/page.tsx

```diff
@@ -32,6 +85,44 @@ const Home: NextPage = () => {
             <span className="font-bold">Checked in builders count:</span>
             <span>{checkedInCounterElement()}</span>
           </p>
+          <div className="flex flex-col text-center text-md">
+            <div>
+              {!isConnectedOptimism ? (
+                <div className="mt-4 text-xl font-bold"> Please Connect to Optimism To Read List of Builders </div>
```

Comment:
No more need this check check https://github.com/BuidlGuidl/batch3.buidlguidl.com/pull/14#issuecomment-1988806853